Helpers/AuthenticationHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿using dotenv.net;
+using OpenQA.Selenium.Support.UI;
+using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+
+public static class AuthenticationHelper
+{
+    static AuthenticationHelper()
+    {
+        // Load environment variables from the .env file
+        DotEnv.Load();
+        Console.WriteLine("Loaded environment variables from .env file.");
+    }
+
+    public static void Login(IWebDriver driver)
+    {
+        // Fetch the username and password from environment variables
+        string username = Environment.GetEnvironmentVariable("MARS_USERNAME") ?? string.Empty;
```

Comment:
Please implement the POM Pattern, please see the below  for example how it looks you can see the pages how they are set 
https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html